std-bits/table/src/main/antlr4/Expression.g4

```diff
@@ -0,0 +1,126 @@
+grammar Expression;
+prog:   expr EOF ;
+
+expr:   expr op=POWER expr                                # Power
+    |   expr op=(MULTIPLY|DIVIDE|MODULO) expr             # MultDivMod
+    |   expr op=(ADD|MINUS) expr                          # AddSub
+    |   expr op=(EQUALS|NOT_EQUALS|LESS_THAN_OR_EQUAL|GREATER_THAN_OR_EQUAL|LESS_THAN|GREATER_THAN) expr  # Compare
+    |   expr (IS_NULL|IS_EMPTY|IS_NOT_EMPTY|IS_NOT_NULL)  # IsNull
+    |   expr (LIKE|NOT_LIKE) expr                         # Like
+    |   expr IN '(' expr (',' expr)* ')'                  # In
+    |   expr (NOT_BETWEEN | BETWEEN) expr AND expr        # Between
+    |   UNARY_NOT expr                                    # UnaryNot
+    |   expr op=AND expr                                  # And
+    |   expr op=OR expr                                   # Or
+    |   IF expr THEN expr ELSE expr END?                  # If
+    |   IDENTIFIER '(' (expr (',' expr)*)? ')'            # Function
```

Comment:
I guess one can familiarize with ANTLR rules, but maybe it would be helpful if we had a comment here stating what kind of functions are permitted here, i.e. `foo()`, `foo(x)`,` foo(x,y)` etc.